pom.xml

```diff
@@ -253,9 +253,9 @@
       when the Hadoop profile is hadoop-2.7, because these are only available in 3.x. Note that,
       as result we have to include the same hadoop-client dependency multiple times in hadoop-2.7.
     -->
-    <hadoop-client-api.artifact>hadoop-client-api</hadoop-client-api.artifact>
```

Comment:
This is open to hear the voices from the reviewers. So don't add some comment here yet.